.github/workflows/playwright_tests_on_vercel.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  deployment_status:
+
+jobs:
+  run-end2endTests:
+    # Only run if the deployment was successful
+    if: github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success'
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the code from the repository
+      - uses: actions/checkout@v2
+
+      # Install project dependencies and Playwright (including browsers)
+      - name: Install dependencies
+        run: npm ci && npx playwright install --with-deps
+
+        # Run Playwright tests
+      - name: Run tests
+        run: npx playwright test --workers=4 --project=chromium
```

Comment:
Not necessary now but we should also run it on Firefox and especially on mobile browsers. This can be a future task though